cmd/flags_node.go

```diff
@@ -34,6 +34,17 @@ var (
 		Usage: "Port for listening incoming api requests",
 		Value: 4050,
 	}
+
+	promiseCurrencyFlag = cli.StringFlag{
+		Name:  "promise.currency",
+		Usage: "Type of currency that will be used for issuing promises",
+		Value: "MYST",
+	}
+	promiseAmountFlag = cli.IntFlag{
+		Name:  "promise.amount",
+		Usage: "Amount of money that will be used for issuing a single promise",
```

Comment:
We should specify format for a user.  I.e: Integer, 1000 == 1 MYST

Reply:
I was thinking that it will be minimal allowed value, as a `wei`

Reply:
Do we need configurable promise at all at this stage.
We agreed to issue `promise` which has full amount from `serviceProposal`